pkg/api/server/register_containers.go

```diff
@@ -153,10 +173,9 @@ func (s *APIServer) RegisterContainersHandlers(r *mux.Router) error {
 	//      "$ref": "#/responses/InternalError"
 	r.HandleFunc(VersionedPath("/containers/{name:..*}/kill"), APIHandler(s.Context, generic.KillContainer)).Methods(http.MethodPost)
 	// swagger:operation GET /containers/{nameOrID}/logs containers LogsFromContainer
-	//
-	// Get logs from container
-	//
 	// ---
+	// summary: Get container logs
+	// description: Get stdout and stderr logs from a container.
```

Comment:
Do we need to mention restriction for support logging drivers? https://docs.docker.com/engine/api/v1.40/#operation/ContainerLogs mentions that only `json-file` and `journald` are supported.